### PR TITLE
operator flux (0.36.0)

### PR DESCRIPTION
**Pull request by paulofrazao**

### CODE REVIEW

File: operators/flux/0.36.0/manifests/flux.v0.36.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,1556 @@
...
+          }
+        }
+      ]
+    capabilities: Seamless Upgrades
+    categories: Integration & Delivery
+    certified: "false"
+    description: Flux is a Continuous Delivery solution for Kubernetes.
+    support: The Flux community
+    operatorframework.io/suggested-namespace: flux-system
+    containerImage: quay.io/redhat-isv-containers/62b4cc4ce987873deb78d87d@sha256:fb607923a6dc9a19bdcf6327f1bedbc9016e4c46fcd5511aa30eb714d9b3e9fc
```
Comment (acornett21) [inline operators/flux/0.36.0/manifests/flux.v0.36.0.clusterserviceversion.yaml:264]: @paulofrazao I think there is some confusing, quay is the `inbound` registry where you push the images to, like how it used to be `scan`. The images are still served to customers from `registry.connect.redhat.com` can you do a diff between the latest certified version and only change the `sha's` of the images here?

**Comment by paulofrazao** [comment]
Hi Adam, thanks for the clarification. I noticed the changes to the
registry and thought we needed to change it in both locations. I will
change it back and give it a try later tonight.

Thanks
-Paulo

File: operators/flux/0.36.0/manifests/helmrelease.crd.yaml [hunk 2]
```diff
@@ -0,0 +1,887 @@
...
+                                - patch
+                                - target
+                              type: object
+                            type: array
+                          patchesStrategicMerge:
+                            description: Strategic merge patches, defined as inline YAML objects.
+                            items:
+                              x-kubernetes-preserve-unknown-fields: true
+                            type: array
+                    type: object
```
Comment (acornett21) [inline operators/flux/0.36.0/manifests/helmrelease.crd.yaml:509]: I'm not sure how these CRD's were generated, but they are not identical to the community ones. Look at the nesting in the community one on the right

image

Comment (acornett21) [reply operators/flux/0.36.0/manifests/helmrelease.crd.yaml:509]:
```
E1102 21:15:12.557300       1 queueinformer_operator.go:290] sync {"update" "flux-system/install-qrx7m"} failed: CustomResourceDefinition.apiextensions.k8s.io "helmreleases.helm.toolkit.fluxcd.io" is invalid: spec.validation.openAPIV3Schema.properties[spec].properties[postRenderers].items.properties[kustomize].type: Required value: must not be empty for specified object fields
```

File: operators/flux/0.36.0/manifests/flux.v0.36.0.clusterserviceversion.yaml (hunk 1 above)
Comment (paulofrazao) [reply operators/flux/0.36.0/manifests/flux.v0.36.0.clusterserviceversion.yaml]: Hi Adam, thanks, yes confusing, partner portal shows quay.io. I just changed it back to registry.connect.redhat.com pointing to new sha256 keys for new images.

**Pull request merged**